content/docs/guides/gke/troubleshooting-gke.md

```diff
@@ -24,6 +24,64 @@ Here are some tips for troubleshooting Cloud IAP.
 
  * Make sure you are using HTTPS
 
+### DNS name not registered
+
+This section provides troubleshooting information for problems creating a DNS entry for your ingress. This section assumes
+you are using Cloud Endpoints and the DNS name
+
+```
+https://${DEPLOYMENT_NAME}.endpoints.${PROJECT}.cloud.goog
+```
+
+Symptoms
+
+  * When you access the the URL in chrome you get the error: **server IP address could not be found**
+  * nslookup for the domain name doesn't return the IP address associated with the ingress
+
+    ```
+    nslookup ${DEPLOYMENT_NAME}.endpoints.${PROJECT}.cloud.goog
+    Server:   127.0.0.1
+    Address:  127.0.0.1#53
+
+    ** server can't find ${DEPLOYMENT_NAME}.endpoints.${PROJECT}.cloud.goog: NXDOMAIN
+    ```
+
+Troubleshooting
+
+1. Check the cloud endpoints resource
+
+   ```
+   kubectl get cloudendpoints -o yaml ${DEPLOYMENT_NAME}
+   kubectl describe cloudendpoints ${DEPLOYMENT_NAME}
+   ```
+
+   * Check if there are errors indicating problems creating the endpoint
+
+1. The status of the cloudendpoints object will contain the cloud operation used to register the operation
+
+   * e.g
```

Comment:
s/e.g/for example:

A tech writer's tip: Always spell out “i.e.” as “that is”, and “e.g.” as “for example”.
Reason: Very many people don't know the difference between "e.g." and "i.e.". The resulting confusion can be quite bad. Also, abbreviations are hard to parse, particularly for people whose first language is other than English.